Hw1/Task3.py

```diff
@@ -0,0 +1,37 @@
+# Задача 3: Вы пользуетесь общественным транспортом? 
+# Вероятно, вы расплачивались за проезд и получали билет с номером. 
+# Счастливым билетом называют такой билет с шестизначным номером, 
+# где сумма первых трех цифр равна сумме последних трех. 
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6. 
+# Вам требуется написать программу, которая проверяет счастливость билета. 
+
+number = input("Введите шестизначный номер вашего билета: ")
+
+check = int(len(number))
+
+
+
+a = int(number[0:3])
```

Comment:
Ноль указывать не обязательно, если срез берется сначала.
```suggestion
a = int(number[:3])
```